lib/Data.php

```diff
@@ -97,10 +106,10 @@ public function send(IEvent $event): int {
 				'object_id' => $event->getObjectId(),
 				'object_name' => $event->getObjectName(),
 				'link' => $event->getLink(),
-			])
-			->execute();
+			]);
+		$this->insertActivity->executeStatement();
```

Comment:
will this work for NC21?

Reply:
Does not matter for this PR, but to remember if this is backported: https://github.com/nextcloud/activity/blob/master/appinfo/info.xml#L38

Reply:
Ah, lovely, thanks!